.github/workflows/wheels.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
           # musllinux arch skip: unlikely someone wants to run dlisio on alpine on non-standard architecture
           # musllinux cp38 skip: latest available numpy doesn't provide musslinux wheels, so it is unlikely useful
           # macosx 38 skip: cibuildwheel can't test it
-          CIBW_SKIP: pp* *-musllinux_i686 *-musllinux_aarch64 cp38*-musllinux* cp38*-macosx_*:arm64 cp36-* cp37-*
+          CIBW_SKIP: pp* *-musllinux_i686 *-musllinux_aarch64 cp38*-musllinux* cp38*-macosx_*arm64 cp36-* cp37-*
```

Comment:
Are we not building and testing on MacOS arm64?

Reply:
Only for python 3.8: as commend above says, "cibuildwheel can't test it"